spec.html

```diff
@@ -19751,95 +19543,17 @@ <h1>Runtime Semantics: Evaluation</h1>
             1. If IsUnresolvableReference(_val_) is *true*, return *"undefined"*.
           1. Set _val_ to ? GetValue(_val_).
           1. [id="step-typeof-web-compat-insertion-point"] NOTE: This step is replaced in section <emu-xref href="#sec-IsHTMLDDA-internal-slot-typeof"></emu-xref>.
```

Comment:
Might as well move this down below the `Assert: _val_ is an Object` step below, and remove the `If _val_ is an Object` from the replacement step.